embulk-core/src/main/java/org/embulk/spi/time/TimestampFormatter.java

```diff
@@ -1,8 +1,6 @@
 package org.embulk.spi.time;
 
 import java.util.Locale;
-import org.joda.time.DateTime;
-import org.joda.time.DateTimeZone;
```

Comment:
NOTE: We'll be introducing `java.time...` here soon. Removing `import org.joda...` and using `org.joda...` directly in the main code so that we won't be confused by `DateTimeZone` (Joda-Time) and `ZoneId` (java.time) in the same place.